.circleci/continue_config.yml

```diff
@@ -317,7 +316,7 @@ jobs:
                 command: |
                   cd services/QuillLMS/client
                   TESTFILES=$(circleci tests glob "app/**/*.{test.*,spec.ts}" | circleci tests split)
-                  node --expose-gc ./node_modules/.bin/jest $TESTFILES --maxWorkers=8 --logHeapUsage
+                  node --expose-gc ./node_modules/.bin/jest $TESTFILES --maxWorkers=2 --logHeapUsage
```

Comment:
reverting to the normal maxWorkers, as this is no longer required after I increased the compute power of our CCI instances